easybuild/easyconfigs/b/BCEL/BCEL-5.2-Java-1.8.eb

```diff
@@ -0,0 +1,33 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'JAR'
+
+name = 'BCEL'
+version = '5.2'
+javaver = '1.8'
+versionsuffix = '-Java-%s' % javaver
```

Comment:
Please use the `javaver` template, which is automatically picked up from the dependencies list (so you can drop line 7):
`versionsuffix = '-Java-%(javaver)s'`